Dockerfile

```diff
@@ -74,10 +71,21 @@ COPY --from=builder /tmp/build/alembic.ini /opt/migrations/alembic.ini
 
 # copy start admin session script
 COPY --from=builder /tmp/build/start_admin_session /usr/sbin/start_admin_session
-RUN chmod 700 /usr/sbin/start_admin_session
+
+# Getting RUN layers together
+RUN gpg --import manufacturing-key.gpg && \
+    rm manufacturing-key.gpg && \
+    chmod 700 /usr/sbin/start_admin_session && \
+    mkdir -p /opt/nebra
 
 # Add python dependencies to PYTHONPATH
 ENV PYTHONPATH="${PYTHON_DEPENDENCIES_DIR}:${PYTHONPATH}"
 ENV PATH="${PYTHON_DEPENDENCIES_DIR}/bin:${PATH}"
 
-ENTRYPOINT ["gunicorn", "--bind", "0.0.0.0:5000", "--timeout", "300", "hw_diag.wsgi:wsgi_app"]
+# Copy environment variables startup script
+COPY setenv.sh /opt/nebra/setenv.sh
```

Comment:
a single COPY will be better. May be 'COPY scripts/*.sh /opt/nebra/'. But just an opinion. You don't actually need to do it.

Reply:
Let's leave it as it is. Maybe someone could add another shell script in the future which should not get into that folder. It would end up there silently. I've always fan of explicit definitions.